interfaces/builtin/unity7.go

```diff
@@ -145,6 +145,12 @@ dbus (receive,send)
     member=Changed
     peer=(label=unconfined),
 
+# url helper
+dbus (send)
+    bus=session
+    interface=com.canonical.UrlLauncher.XdgOpen
```

Comment:
Done, thanks! I also updated https://github.com/ubuntu-core/snap-desktop-links/commit/21e9c2bff2ba64d39f53e29e0179565171657bee and https://launchpadlibrarian.net/260686704/livecd-rootfs_2.411~ppa3_2.411~ppa4.diff.gz